### DISCUSSION TITLE
Current line background

### DISCUSSION THREAD

**Discussion by dxps** [dbeaver/dbeaver#12936] [Q&A] [answered]
Dear all,

Can somebody provide some help, please? I used this tool and I'd like to keep using and promoting it.
In a new Linux distro setup and with a new DBeaver release, I just discovered that I cannot set the current line bg.

**Context:**
- using DBeaver version `21.0.1.202103211229`
- with _Theme:_ `System` and _Color and Font theme:_ `Default (current)`

**Issue**
- As you can see in the screenshot below, for now it all looks great, but the current line looks too brightly.
- How can the current line background color be customized?

**Describe the solution you'd like**
Have an option in _Preferences > User Interface > Color and Fonts_ menu to customize it.

![image](https://user-images.githubusercontent.com/14140226/113507144-64257c80-9551-11eb-9c1b-2e003d285a98.png)

Thank you!

**Comment by kseniiaguzeeva** [comment]
We will implement it in 21.1.2.

**Comment by flambchap** [comment, marked as answer]
Actually, there is nothing to implement. Go to `Preferences` -> `Editors` -> `Text Editors`. Under `Appearance color options`, choose `Current line highlight` and set the desired color.
